docs/content/concepts/webserver/ui.mdx

```diff
@@ -61,15 +61,45 @@ height={1010}
 
 - **Description**: The **Asset catalog** page lists all [assets](/concepts/assets/software-defined-assets) in your Dagster deployment, which can be filtered by asset key and/or [asset group](/concepts/assets/software-defined-assets#grouping-assets). Clicking an asset opens the [**Asset details** page](#asset-details) for that asset. You can also navigate to the [Global asset lineage](#global-asset-lineage) page, reload definitions, and materialize assets.
 
+  **If using Dagster Cloud Pro**, you have the option to use a new version of this page. Click the **Asset catalog (Cloud Pro)** tab for more information.
+
 - **Accessed by:** Clicking **Assets** in the top navigation bar
 
 <!-- ![The Asset Catalog page in the Dagster UI](/images/concepts/webserver/asset-catalog.png) -->
 
 <Image
 alt="The Asset Catalog page in the Dagster UI"
 src="/images/concepts/webserver/asset-catalog.png"
-width={3572}
-height={1944}
+width={2640}
+height={1514}
+/>
+
+</TabItem>
+<TabItem name="Asset catalog (Cloud Pro)">
+
+### Asset catalog (Cloud Pro)
+
+<Note>A Dagster Cloud Pro plan is required to use this feature.</Note>
+
+- **Description**: This **experimental** version of the **Asset catalog** page includes all the information and functionality of the original page, broken out by compute kind, asset group, code location, etc. On this page, you can:
+
+  - View all [assets](/concepts/assets/software-defined-assets) in your Dagster deployment
+  - View [details](#asset-details) about a specific asset by clicking on it
+  - Search assets by asset key, group, compute kind, code location, etc.
+  - Access the [Global asset lineage](#global-asset-lineage)
+  - Reload definitions
+
+- **Accessed by:**
+  1. Enabling the **Use new asset overview page** setting in your [user settings](/concepts/webserver/ui-user-settings)
```

Comment:
This flag is now on by default, defer to you to decide whether that's worth mentioning or not